Common/PkmnModels/Stats.cs

```diff
@@ -0,0 +1,26 @@
+using System.Runtime.Serialization;
+
+namespace Common.PkmnModels
+{
+    [DataContract]
+    public enum Stat
+    {
+        [EnumMember(Value = "hp")] Hp,
+        [EnumMember(Value = "atk")] Atk,
+        [EnumMember(Value = "def")] Def,
+        [EnumMember(Value = "spA")] SpA,
+        [EnumMember(Value = "spD")] SpD,
+        [EnumMember(Value = "spe")] Spe,
+    }
+
+    [DataContract]
+    public struct Stats
+    {
+        [DataMember(Name = "hp")] public int Hp { get; set; }
+        [DataMember(Name = "atk")] public int Atk { get; set; }
+        [DataMember(Name = "def")] public int Def { get; set; }
+        [DataMember(Name = "spA")] public int SpA { get; set; }
+        [DataMember(Name = "spD")] public int SpD { get; set; }
```

Comment:
How breaking of a change would it be to make these abbreviations all-lowercase? For reference, Showdown uses lowercase keys in stat tables.

Reply:
what does ps call them? just the same but lowercase?

Reply:
yes

Reply:
a quick fulltext search on old core, pbrengine and pokecat shows:
- pokesets would need to be converted, or alternatively stay inconsistent with the ps convention and be converted on instantiation
- code in pokecat needs to be adjusted
- data files need to be adjusted (e.g. pokedex.json)
- documentation needs to be adjusted (examples, specification)
- some injection code in pbrengine needs to be adjusted
- some overlay code needs to be adjusted
- some matchmaker code needs to be adjusted
- API needs to be adjusted (publicly facing breaking change)

Reply:
forgot something:
- already persisted data needs to be adjusted, or a change in the data at some point in time needs to be taken into consideration (document it in the api? convert on the fly?)

Reply:
I can't really see any benefit of making this change...